packages/vuetify/src/components/VDataIterator/VDataIterator.tsx

```diff
@@ -69,6 +76,12 @@ export const makeVDataIteratorProps = propsFactory({
   ...makeDataTableGroupProps(),
   ...makeFilterProps(),
   ...makeTagProps(),
+  ...makeTransitionProps({
+    transition: {
+      component: VSlideYTransition as Component,
+      hideOnLeave: true
```

Comment:
```suggestion
      hideOnLeave: true,
```

Reply:
suggestion: Setup auto-formatting with DPrint || Prettier to auto-format stylistic things like this.

Reply:
We do, but the user or their IDE has to run it.